src/components/FooterComponent/FooterLinks/footerLinksTypes.ts

```diff
@@ -0,0 +1,6 @@
+interface IFooterLinksTypes{
```

Comment:
you can use export here like export interface... and don't write additional code (line 6 e.g.)